src/components/DeleteComment.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from "react";
+import { deleteComment } from "../utils/API";
+
+const DeleteComment = ({ comment_id, comments, setComments }) => {
+  const [isDeleting, setIsDeleting] = useState(false);
+
+  const handleClick = () => {
+    setIsDeleting(true);
+    setComments((currComments) => {
+      return currComments.filter(
+        (comment) => comment.comment_id !== comment_id
+      );
+    });
+    alert("Comment has been deleted!");
```

Comment:
Just so you know `alert`'s are synchronous and blocking. Your whole app will freeze whilst it's open which is generally not a good thing. It's fine here but be careful of them in the real world.